src/Storefront/Page/GenericPageLoader.php

```diff
@@ -63,6 +63,7 @@ public function __construct(
      * @throws CategoryNotFoundException
      * @throws InconsistentCriteriaIdsException
      * @throws MissingRequestParameterException
+     * @throws \Shopware\Core\Framework\DataAbstractionLayer\Exception\InconsistentCriteriaIdsException
```

Comment:
only annotate exceptions thrown in the method 😉

Reply:
Damn auto-exceptions :D